nativescript-angular/platform-common.ts

```diff
@@ -210,7 +210,7 @@ export class NativeScriptPlatformRef extends PlatformRef {
     }
 
     @profile
-    private livesync() {
+    public livesync() {
```

Comment:
Should this be public livesync or _livesync?

Reply:
Really not sure. @vakrilov what do you think? If it is lower case _livesync no need to document it?

Reply:
I think it is better to make it `public _livesync()` for now